### PR TITLE
[CN-668] - Sync Hazelcast Operator repo with Hazelcast Charts repo

### PR DESCRIPTION
**Pull request by cheels**
## Description

Added a new workflow that allows pushing helm chart directly into https://github.com/hazelcast/charts.git based on changes in helm-chart/ folder in current repo

### CODE REVIEW

File: .github/workflows/helm-chart-release.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Helm Chart Release
+
+on:
+  push:
+    paths:
+      - 'helm-chart/**'
```
Comment (cagric0) [inline .github/workflows/helm-chart-release.yaml:6]:
```suggestion
      - 'helm-charts/**'
```

File: .github/workflows/helm-chart-release.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        with:
+          token: ${{ secrets.DEVOPS_GITHUB_TOKEN }}
+
+      - name: Commit And Push Changes To 'Hazelcast Charts' Repository
+        run: |
+          echo ${{ secrets.DEVOPS_GITHUB_TOKEN }} | gh auth login --with-token
+          git config user.email "devopshelm@hazelcast.com"
+          git config user.name "devOpsHelm"
+          git subtree add --prefix=charts https://github.com/hazelcast/charts.git master --squash
+          cp -rf helm-chart/hazelcast-platform-operator/* charts/stable/hazelcast-platform-operator
```
Comment (cagric0) [inline .github/workflows/helm-chart-release.yaml:23]:
```suggestion
          cp -rf helm-charts/hazelcast-platform-operator/* charts/stable/hazelcast-platform-operator
```

File: .github/workflows/helm-chart-release.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+          git add charts/
+          git commit --signoff -m "${{github.event.head_commit.message}}"
```
Comment (cagric0) [inline .github/workflows/helm-chart-release.yaml:25]: Is this `github.event.head_commit.message` the commit message from operator repo?

**Review by cagric0 — CHANGES_REQUESTED** [review]

File: .github/workflows/helm-chart-release.yaml (hunk 3 above)
Comment (cheels) [reply .github/workflows/helm-chart-release.yaml:25]: Yes, it comes from the operator repo

**Review by cagric0 — APPROVED** [review]
It's not a big deal but it seems the PR name is not proper. Could you update it?

**Review by dzeromski-hazelcast — APPROVED** [review]

File: .github/workflows/helm-chart-release.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+name: Helm Chart Release
```
Comment (cheels) [inline .github/workflows/helm-chart-release.yaml:1]:
```suggestion
name: Helm Chart Update
```

**Pull request merged**